tests/test_impactgraph_api.py

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of INSPIRE.
+# Copyright (C) 2016 CERN.
+#
+# INSPIRE is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# INSPIRE is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with INSPIRE. If not, see <http://www.gnu.org/licenses/>.
+#
+# In applying this license, CERN does not waive the privileges and immunities
+# granted to it by virtue of its status as an Intergovernmental Organization
+# or submit itself to any jurisdiction.
+
+"""Tests for Impact Graph API."""
+
+import json
+import os
+import pkg_resources
+
+from invenio.testsuite import InvenioTestCase
```

Comment:
> Both works, invenio.testsuite is aliased to invenio_testsuite.

You mean `invenio_testing`?

I remember in the past `invenio_testing` did not have all the functionality ported so we were using `invenio.testuite` instead - which is this one https://github.com/inspirehep/invenio/tree/labs/invenio/testsuite